pennylane/init.py

```diff
@@ -183,321 +131,643 @@ def random_layers_normal(n_layers, n_wires, n_rots=None, mean=0, std=0.1, seed=N
     """
     if seed is not None:
         np.random.seed(seed)
-
     n_if = n_wires * (n_wires - 1) // 2
 
-    theta_1 = np.random.uniform(low=low, high=high, size=(n_if, ))
-    phi_1 = np.random.uniform(low=low, high=high, size=(n_if, ))
-    varphi_1 = np.random.uniform(low=low, high=high, size=(n_wires,))
-    r = np.random.normal(loc=mean_active, scale=std_active, size=(n_wires,))
-    phi_r = np.random.uniform(low=low, high=high, size=(n_wires,))
-    theta_2 = np.random.uniform(low=low, high=high, size=(n_if, ))
-    phi_2 = np.random.uniform(low=low, high=high, size=(n_if, ))
-    varphi_2 = np.random.uniform(low=low, high=high, size=(n_wires,))
-    a = np.random.normal(loc=mean_active, scale=std_active, size=(n_wires,))
-    phi_a = np.random.uniform(low=low, high=high, size=(n_wires,))
-    k = np.random.normal(loc=mean_active, scale=std_active, size=(n_wires,))
-
-    return [theta_1, phi_1, varphi_1, r, phi_r, theta_2, phi_2, varphi_2, a, phi_a, k]
-
+    theta = np.random.uniform(low=low, high=high, size=(n_if,))
+    return theta
 
-def cvqnn_layer_normal(n_wires, mean=0, std=1, mean_active=0, std_active=0.1, seed=None):
-    r"""Creates a list of eleven parameter arrays for :func:`~.CVNeuralNetLayer`, where both active and non-active
-    gate parameters are drawn from normal distributions.
 
-    The shape of the arrays is ``(n_wires*(n_wires-1)/2)`` for the parameters used in an interferometer,
-    and ``(n_wires)``  else.
+def interferometer_phi_uniform(n_wires, low=0, high=2*pi, seed=None):
+    r"""Creates a parameter array for the ``phi`` input of :func:`Interferometer`, drawn from a uniform
+    distribution.
 
-    All gate parameters are drawn from a normal distribution with mean ``mean`` and standard deviation ``std``,
-    except from the three types of 'active gate parameters': the displacement amplitude, squeezing amplitude and kerr
-    parameter. These active gate parameters are sampled from a normal distribution with mean ``mean_active`` and
-    standard deviation ``std_active``. Since they influence the mean photon number (or energy) of the quantum system,
-    one typically wants to initialize them with values close to zero.
+    The array has shape ``(n_wires*(n_wires-1)/2, )``.
 
     Args:
-        n_wires (int): number of modes of the CV Neural Net
+        n_wires (int): number of modes that the interferometer acts on
 
     Keyword Args:
-        mean (float): mean of non-active parameters
-        std (float): standard deviation of non-active parameters
-        mean_active (float): mean of active gate parameters
-        std_active (float): standard deviation of active gate parameters
+        low (float): minimum value of uniform distribution
+        high (float): maximum value of uniform distribution
         seed (int): seed used in sampling the parameters, makes function call deterministic
 
     Returns:
-        list of parameter arrays
+        list: list of parameter arrays
```

Comment:
How come changed here but not in `interferometer_all` or `interferometer_theta_uniform`?

Reply:
In fact, the docstrings of these functions each say they create `a parameter array`. @mariaschuld can you double check?